Content.Server/Traits/TraitSystem.Functions.cs

```diff
@@ -297,16 +297,52 @@ public override void OnPlayerSpawn(EntityUid uid,
 public sealed partial class TraitVVEdit : TraitFunction
 {
     [DataField, AlwaysPushInheritance]
-    public Dictionary<string, string> VVEdit { get; private set; } = new();
+    public Dictionary<string, string> Changes { get; private set; } = new();
 
     public override void OnPlayerSpawn(EntityUid uid,
         IComponentFactory factory,
         IEntityManager entityManager,
         ISerializationManager serializationManager)
     {
         var vvm = IoCManager.Resolve<IViewVariablesManager>();
-        foreach (var (path, value) in VVEdit)
-            vvm.WritePath(path, value);
+        string idPath;
+        foreach (var (path, value) in Changes)
+        {
+            idPath = path.Replace("$ID", uid.ToString());
+            vvm.WritePath(idPath, value);
+        }
```

Comment:
```suggestion
        foreach (var (path, value) in Changes)
        {
            var idPath = (string) path.Replace("$ID", uid.ToString());
            if (!string.TryParse(vvm.ReadPathSerialized(idPath), out var currentValue))
                continue;
            vvm.WritePath(idPath, value);
        }
```
You can try it without the explicit cast as a (string), but this would fix the compiler error you had with it not intuiting that path was factually a string.

Reply:
Done